10up/windows-azure-storage
# fix window title after image uploaded

### Description of the Change
This PR fixes an issue with the media uploader title stuck at "upload to Azure..." even after the image is uploaded. The title remains there even if you look at other images

Closes #86 

### How to test the Change
- Pull down the branch
- Go to Dashboard > Media
- Click on "Add new" 
- Select an image to upload
- While is uploading click in the thumbnail to open the image windows
- Check at the title of the image says: "Uploading to Azure..."
- When the image is uploaded, the title should change to "Attachment details" (Default title)

### Changelog Entry
> Fixed - Bug fix
Fix media uploader title "Uploading to Azure..." stuck after image is uploaded


### Credits
Props @hugosolar, @eflorea (For reporting)


### Checklist:
